check.go

```diff
@@ -113,7 +113,7 @@ type CheckCreateInput struct {
 	Level    graphql.ID    `json:"levelId"`
 	Owner    *graphql.ID   `json:"ownerId,omitempty"`
 	Filter   *graphql.ID   `json:"filterId,omitempty"`
-	Notes    string        `json:"notes,omitempty"`
+	Notes    *string       `json:"notes"`
```

Comment:
no need for a pointer here - and we should keep the `omitemtpy` since this is on the "create" side.  The "update" is what the problem is.